Resources/ServerInfo/_Moffstation/Guidebook/ServerRules/MoffR2Metagaming.xml

```diff
@@ -0,0 +1,15 @@
+<Document>
+  # Section 2: Metagaming
+
+  2.1 Do not metagame (i.e. using or sharing information that your character has no way of knowing or obtaining, such as knowledge on the exact loadouts of antagonists, or things seen or heard while dead).
+
+  2.2 Do not metacommunicate or metacomm (communicate with other players via a method external to the game) with other players in the same game server. This includes sitting in VC muted/deafened or having active text chats with other members. If you need to say something Out Of Character (OOC), please do so in game via the Local Out Of Character (LOOC).
+
+  2.3 Do not metagrudge nor metafriend. That is, treating a character unfairly solely based on the out-of-character relationship with the player; or treating them unfairly beyond acceptable and reasonable conduct - for example, attacking on sight - because of in-character interactions in previous rounds.
+
+  2.4 Players are allowed to retain and share knowledge of past experiences and prior shifts. However, treating a character unfairly solely based on happenings outside of the current round, is forbidden (for example, treating someone with more or less suspicion or trust solely based on how often they are an antagonist).
```

Comment:
I think some clarification that knowledge that breaches the metashield is not retained. Any removed metashields only persist for that shift.